GothicVRProject/GothicVR
# Feature/modularization

# To test:
1. Checkout branch: feature/modularization
2. See image 1: Install HVR (Assets/HurricaneVR/*) and Activate it via Menu in Unity
3. See image 2: Skip main menu in feature flags (not working as of now).

Then the game should load and place you. You can now start testing.

Known issues:
* The main menu isn't working with HVR so far. I think it's related to the fact, that the HVRPlayer has no menu set up so far
* When you enter the game, it always spawns you at 0,0,0 (So lean a little forward when hitting "play". Otherwise the character will fall through the ground


Helpful hints:
* See image 3: You can always check if you're currently using HVR or OXR.
* When you want to implement main menu etc. Just hit "Deactivate HVR" in the same menu. Then the game boots with OXR logic and you can compare
* Image 4: The custom HVR adapter logic and OXR adapter logic is now in a separate module

![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/d226ee08-f146-45a4-911b-7fd9a9ffa75f)

![image](https://github.com/GothicVRProject/GothicVR/assets/120568393/7ba09515-8696-4380-8dca-5390082ddb04)
